src/Symfony/Component/Security/Http/RateLimiter/DefaultLoginRateLimiter.php

```diff
@@ -41,7 +41,7 @@ protected function getLimiters(Request $request): array
     {
         return [
             $this->globalFactory->create($request->getClientIp()),
-            $this->localFactory->create(strtolower($request->attributes->get(Security::LAST_USERNAME)).'-'.$request->getClientIp()),
+            $this->localFactory->create(mb_strtolower($request->attributes->get(Security::LAST_USERNAME)).'-'.$request->getClientIp()),
```

Comment:
we should not rely on the global php.ini config
this should instead be:
`\function_exists('mb_strtolower') && preg_match('//u', $v) ? mb_strtolower($v, 'UTF-8') : strtolower($v);`

Reply:
Do we need `function_exists('mb_strtolower')`? The polyfill should make sure that the method exists, shouldn't it?

Reply:
Ok done.. I kept the preg_match() guard, not sure if it's really needed or not? Is the polyfill not enough?